packages/react-ui-components/src/components/SchemaForm/Templates/BaseInputTemplate/BaseInputTemplate.tsx

```diff
@@ -0,0 +1,64 @@
+import { EInputFieldType, EValidationState } from '@kelvininc/ui-components';
+import classNames from 'classnames';
+import { get, isArray, isEmpty } from 'lodash-es';
+import React, { useCallback, useMemo } from 'react';
+import { KvTextField } from '../../../stencil-generated';
+import styles from './BaseInputTemplate.module.scss';
+import { WidgetProps } from '@rjsf/utils';
+import { INPUT_TYPES } from './BaseInputTemplate.config';
+import { JSONSchema7TypeName } from 'json-schema';
+
+const getInputType = (type?: JSONSchema7TypeName | JSONSchema7TypeName[]) => (type && !isArray(type) ? INPUT_TYPES[type] ?? EInputFieldType.Text : EInputFieldType.Text);
+
+const BaseInputTemplate = ({
+	id,
+	placeholder,
+	required,
+	readonly,
+	disabled,
+	label,
+	value,
+	onChange,
+	onBlur,
+	autofocus,
+	options,
+	schema,
+	rawErrors = [],
+	uiSchema,
+	type
+}: WidgetProps) => {
+	const _onChange = useCallback((value: CustomEvent<string>) => onChange(value?.detail ? value.detail : options.emptyValue), [onChange, options]);
```

Comment:
Remove the underscore from your callbacks

Reply:
```suggestion
	const _onChange = useCallback((value: CustomEvent<string>) => onChange(value?.detail ?? options.emptyValue), [onChange, options]);
```

Reply:
if remove I have conflict with `onChange` that comes from props

Reply:
Hum, correct, you could change this to `onTextChange`, but for me, it's ok to keep it like this 👍